HexLib is a missing dependency in mvn

With the default `pom.xml` I get the following error:

> [ERROR] /C:/Users/tschoening/Downloads/kaitai_struct_gui-master/src/main/java/io/kaitai/struct/visualizer/VisualizerPanel.java:[3,25] package at.HexLib.library does not exist

For test purposes, I added the local lib provided in the repo as a dependency and that worked:

		<dependency>
			<groupId>at.HexLib</groupId>
			<artifactId>HexLib</artifactId>
			<version>0.0.0</version>
			<scope>system</scope>
			<systemPath>${project.basedir}/lib/HexLib.jar</systemPath>
		</dependency>

Is this dependency simply missing or did you have something different in mind how this should work in Maven?

--- a/pom.xml
+++ b/pom.xml
@@ -19,6 +19,13 @@
             <artifactId>kaitai-struct-compiler_2.11</artifactId>
             <version>0.5-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>at.HexLib</groupId>
+            <artifactId>HexLib</artifactId>
+            <version>0.0.0</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/lib/HexLib.jar</systemPath>
+        </dependency>
     </dependencies>
 
     <build>

